testkit/src/main/java/org/apache/calcite/test/catalog/MockCatalogReaderSimple.java

```diff
@@ -86,6 +89,12 @@ private void registerTableEmp(MockTable empTable, Fixture fixture) {
     empTable.addColumn("COMM", fixture.intType);
     empTable.addColumn("DEPTNO", fixture.intType);
     empTable.addColumn("SLACKER", fixture.booleanType);
+    empTable.setReferentialConstraints(
+        Lists.newArrayList(
+            RelReferentialConstraintImpl.of(
+        Lists.newArrayList(DEFAULT_CATALOG, DEFAULT_SCHEMA, "EMP"),
+        Lists.newArrayList(DEFAULT_CATALOG, DEFAULT_SCHEMA, "DEPT"),
+        Lists.newArrayList(IntPair.of(7, 0)))));
```

Comment:
Might help to state what columns are linked for clarity (in a comment)

Reply:
Add the comment.